integration_tests/run_pyspark_from_build.sh

```diff
@@ -268,6 +268,10 @@ else
         export PYSP_TEST_spark_driver_memory="${DRIVER_MEMORY}"
     fi
 
+    # TODO: https://github.com/NVIDIA/spark-rapids/issues/10940
+    EXECUTOR_MEMORY=${EXECUTOR_MEMORY:-"1536m"}
+    export PYSP_TEST_spark_executor_memory=${PYSP_TEST_spark_executor_memory:-"${EXECUTOR_MEMORY}"}
```

Comment:
Most of the tests run  Spark in the local mode. If spark master is local we need spark.driver.memory

Reply:
I think for local mode, spark would combine driver+executor memory request, no?

Reply:
Did a quick verification in spark320. I was wrong, executor memory was ignored in local mode at least for spark 320. let me update the code, thanks